client/client.go

```
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"os"
+)
+
+type Client struct {
+	Config *Config
+}
+
+func NewClient(c *Config) *Client {
+	s := &Client{
+		Config: c,
+	}
+
+	return s
+}
+
+func (s *Client) Start() {
+
+	raddr, err := net.ResolveTCPAddr(s.Config.Type, fmt.Sprintf("%s:%d", s.Config.Host, s.Config.Port))
+	if err != nil {
+		log.Fatal("ResolveTCPAddr error: ", err)
+	}
+
+	conn, err := net.DialTCP(s.Config.Type, nil, raddr)
+	if err != nil {
+		log.Fatal("DialTCP error: ", err)
+	}
+
+	defer func() {
+		_ = conn.Close()
+	}()
+
+	log.Printf("Connected to %s:%d", s.Config.Host, s.Config.Port)
+
+	fmt.Println("Type your message:")
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print(">: ")
+		rs, _ := reader.ReadString('\n')
```

Why do you ignore an error ?